src/west/cmd/project.py

```diff
@@ -350,7 +349,6 @@ def _all_projects(args):
 
     _validate_manifest(manifest_path)
 
-
```

Comment:
Maybe we can agree on a flake8 configuration file and put it into CI? I tend to keep it at its default settings, but I'm happy to customize if it improves readability.

Reply:
Could just remove them for now. Might not be important enough to muck around with style checkers. I don't have a perfectly consistent style otherwise either. ;)